flag.go

```diff
@@ -10,6 +11,10 @@ import (
 	"time"
 )
 
+var (
+	slPfx = fmt.Sprintf("sl:::%d:::", time.Now().UTC().UnixNano())
```

Comment:
Gotcha, that makes sense to me, do you mind adding a comment to that effect for _enhanced_ clarity?